tooling/redhat-camel-spring-boot-bom/pom.xml

```diff
@@ -90,6 +90,12 @@
                 <artifactId>undertow-servlet</artifactId>
                 <version>2.3.7.Final</version>
             </dependency>
+            <!-- Overrides snappy-java -->
+            <dependency>
+                <groupId>org.xerial.snappy</groupId>
+                <artifactId>snappy-java</artifactId>
+                <version>${snappy-java-version}</version>
```

Comment:
is this place holder correct? shouldn't it be resolved with the actual version?

Reply:
I'm a little confused what the question here is but this is what we did in 3.20.1 to fix it : 
https://github.com/jboss-fuse/camel-spring-boot/blob/camel-spring-boot-3.20.1-branch/tooling/redhat-camel-spring-boot-bom-generator/src/main/resources/pom.xml#L93-L98

If we don't do this, spring-boot brings avro's version commons-compress through the jackson bom.

Reply:
I'm going to further bullet-proof this with https://github.com/jboss-fuse/camel-spring-boot/pull/273 - I still think we should override snappy-java though

Reply:
ok!